src/autorest-core/app.ts

```diff
@@ -285,7 +285,18 @@ async function currentMain(autorestArgs: string[]): Promise<number> {
   subscribeMessages(api, () => exitcode++);
   const artifacts: Artifact[] = [];
   const clearFolders: string[] = [];
-  api.GeneratedFile.Subscribe((_, artifact) => artifacts.push(artifact));
+  api.GeneratedFile.Subscribe((_, artifact) => {
+    if (artifact.uri.startsWith("stdout://")) {
+      config.Message({
```

Comment:
note to tomorrow Johannes: why didn't you just put that in the emitter, as the message even suggests?